src/AttachVS/AttachVs.cs

```diff
@@ -215,18 +218,18 @@ private static bool IsVsOrNull(Process process)
         return isVs;
     }
 
-    private static bool IsCorrectParent(Process currentProcess, Process parent)
+    private static bool IsCorrectParent(Process currentProcess, Process? parent)
```

Comment:
We could also ensure that `parent` is not nullable and handle potential nulls on calling sites. Any preference?